documentation/src/main/resources/pages/ditto/connectivity-mapping.md

```diff
@@ -350,6 +351,87 @@ Example configuration:
   }
 }
 ```
+### CloudEvents Mapper
+
+This mapper maps incoming [CloudEvent](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md) to Ditto Protocol. It provides support for both Binary CloudEvents as well as Structured CloudEvents.
+
+**Note**: The mapper only supports incoming messages with `content-type:application/cloudevents+json`. Messages that do not conform to this are ignored.
+
+#### Specifications
```

Comment:
That's a bad heading - you provide examples below this heading and not "Specifications".
```suggestion
#### CloudEvents examples
```